packages/trader/src/App/Components/Elements/ContractAudit/contract-details.tsx

```diff
@@ -1,19 +1,21 @@
 import React from 'react';
 import { Money, Icon, ThemedScrollbars } from '@deriv/components';
-import { localize } from '@deriv/translations';
+import { localize, Localize } from '@deriv/translations';
```

Comment:
may as well change to Localize component all together

Reply:
Some strings (wrapped with `localize()`) are used inside of `<Localize />` as a variables. If I get rid of `localize()` and replaced each place with `<Localize />`, then it wouldn't work (`<Localize />` inside `<Localize />`)